account/src/helpers/password.js

```diff
@@ -0,0 +1,11 @@
+import bcrypt from 'bcryptjs';
+
+export async function hashPassword(password) {
+    const hashedPassword = await bcrypt.hash(password, parseInt(process.env.HASH_SALT));
```

Comment:
i think the right approach is to create a variable with the result before and put the variable here after